PocketKit/Sources/PocketKit/Item/ReadableHostViewController.swift

```diff
@@ -99,7 +91,40 @@ class ReadableHostViewController: UIViewController {
 
     @objc
     private func archiveArticle() {
-        readableViewModel.archiveArticle()
+        readableViewModel.archive()
+    }
+
+    @objc
+    private func unarchiveArticle() {
+        readableViewModel.unarchive { [weak self] success in
+            if success, let archiveButton = self?.getArchiveButton() {
+                self?.navigationItem.rightBarButtonItems?[2] = archiveButton
+            }
+        }
+    }
+
+    private func getArchiveButton() -> UIBarButtonItem {
```

Comment:
there are multiple options for doing something like this -- another is `lazy var`. something we could talk about as a team to determine if there's a pattern we should follow.

Reply:
Ref: https://pocket.slack.com/archives/C01UH57EJKD/p1680188131450979